include/cutlass/gemm/kernel/xe_gemm.hpp

```diff
@@ -226,9 +230,16 @@ class GemmUniversal<
     // Get the appropriate blocks for this sub_group -- potential for sub_group locality
     int thread_idx = int(ThreadIdxX());
     auto blk_shape = TileShape{};
-    auto m_coord = BlockIdxY();
-    auto n_coord = BlockIdxX();
-    auto l_coord = BlockIdxZ();
+    int m_coord, n_coord, l_coord;
+    if (params.scheduler.raster_order_ == TileScheduler::RasterOrder::AlongN) {
+      m_coord = BlockIdxY();
+      n_coord = BlockIdxX();
+      l_coord = BlockIdxZ();
+    } else {
+      m_coord = BlockIdxX();
+      n_coord = BlockIdxY();
+      l_coord = BlockIdxZ();
+    }
```

Comment:
I think we can remove this `if` condition since we agreed not to have index switching.

```suggestion
    auto m_coord = BlockIdxY();
    auto n_coord = BlockIdxX();
    auto l_coord = BlockIdxZ();
```

Reply:
@muhammad-tanvir-1211  if we are keeping this can you check if the order is correct and the default is `AlongN`?